client/examples/slack-bot/.gitignore

```diff
@@ -0,0 +1,35 @@
+lib-cov
```

Comment:
In general, I think that it'd be better to rename the directory from 'slack-bot' to `magic-money-bot` (or something a bit less prudent), as we might have other slack bots here, etc.